src/main/java/org/dhbw/mosbach/ai/Bearing.java

```diff
@@ -0,0 +1,102 @@
+package org.dhbw.mosbach.ai;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Bearing {
+    private double cdyn;
+    private double y;
+    private double e;
+    private double xB1;
+    private double p;
+    private double fr;
+    private double fa;
+    private double lh10; 
+
+    public Bearing(double cdyn, double y, double e, double xB1) {
+        this.cdyn = cdyn;
+        this.y = y;
+        this.e = e;
+        this.xB1 = xB1;
+    }
+
+    @JsonProperty
+    public double getCdyn() {
```

Comment:
There is a lot of boilerplate code in your classes. Please take a lot a this [book](https://github.com/philippabele/ball-bearings-with-quarkus/blob/8c2ea4f963d0833194a82485d4542643b74f89f8/docs/Bibliography.bib#L44) on page 132 (not PDF page) and extend your class with PanacheEntity.

May this [example](https://quarkus.io/guides/hibernate-orm-panache#first-an-example) helps you as well.